example/src/Examples.tsx

```diff
@@ -3,14 +3,15 @@ import {Text, View, StyleSheet} from 'react-native';
 import Slider, {SliderProps} from '@react-native-community/slider';
 
 const SliderExample = (props: SliderProps) => {
-  const [value, setValue] = useState(0);
+  const [value, setValue] = useState(props.value ?? 0);
   return (
     <View>
       <Text style={styles.text}>{value && +value.toFixed(3)}</Text>
       <Slider
         step={0.5}
         style={styles.slider}
         {...props}
+        value={value}
```

Comment:
What if we leave the `value` as default?

Reply:
This would certainly work, but it feels wrong to me for the slider to not be a controlled component. What do you think?